joplin/templates/joplin_UI/primary_navigation.html

```diff
@@ -0,0 +1,31 @@
+{% load staticfiles wagtailadmin_tags %}
+{% load i18n %}
+
+{% block content %}
+
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Medium+IBM+Plex+Sans&display=swap" rel="stylesheet">
```

Comment:
similar to above, any particular reason not to include this on base.html for the admin? Are there some places where we don't expect to need to access this typeface?

Reply:
No, I'm gonna move it. see above comment.